nixos/modules/services/audio/snapserver.nix

```diff
@@ -30,43 +27,43 @@ let
   streamToOption = name: opt:
     let
       os = val:
-        optionalString (val != null) "${val}";
+        lib.optionalString (val != null) "${val}";
       os' = prefix: val:
-        optionalString (val != null) (prefix + "${val}");
+        lib.optionalString (val != null) (prefix + "${val}");
       flatten = key: value:
         "&${key}=${value}";
     in
       "--stream.stream=\"${opt.type}://" + os opt.location + "?" + os' "name=" name
         + os' "&sampleformat=" opt.sampleFormat + os' "&codec=" opt.codec
-        + concatStrings (mapAttrsToList flatten opt.query) + "\"";
+        + lib.concatStrings (lib.mapAttrsToList lib.flatten opt.query) + "\"";
```

Comment:
This is a regression, it was supposed to be the flatten from the let expression over on line 33.

Reply:
Thanks for the alert. I've opened https://github.com/NixOS/nixpkgs/pull/344243 to resolve this issue.